cmd/dep/status_test.go

```diff
@@ -150,6 +157,19 @@ func TestBasicLine(t *testing.T) {
 					t.Errorf("Did not find expected Table status: \n\t(GOT) %v \n\t(WNT) %v", buf.String(), wantStatus)
 				}
 			}
+
+			buf.Reset()
+			template, _ := template.New("status").Parse("PR:{{.ProjectRoot}}, Const:{{.Constraint}}, Ver:{{.Version}}, Rev:{{.Revision}}, Lat:{{.Latest}}, PkgCt:{{.PackageCount}}")
+			templateout := &templateOutput{w: &buf, tmpl: template}
+			templateout.BasicHeader()
+			templateout.BasicLine(&test.status)
+			templateout.BasicFooter()
+
+			for _, wantStatus := range test.wantTemplateStatus {
+				if ok := strings.Contains(buf.String(), wantStatus); !ok {
+					t.Errorf("Did not find expected template status: \n\t(GOT) %v \n\t(WNT) %v", buf.String(), wantStatus)
+				}
+			}
```

Comment:
This test is good, but it doesn't actually ensure that these types remain of type string, or more importantly that templates using the `eq` function continue to work which is sorta what this whole patch is about. 

I'd recommend having another test (or another chunk in this test) that attempts equality on every field.  This may not even need an expected output, but it should at least ensure that the template can be executed without errors.